### DISCUSSION TITLE
Curso.Prep.Henry/02-JS-I/tarea/ homework.js

### DISCUSSION THREAD

**Discussion by Leonardo2696** [atralice/Curso.Prep.Henry#161] [Q&A]
hola buenas, tengo problemas para resolver los siguientes problemas si alguien me pudiera explicar c los agradecería

function combinarNombres(nombre, apellido) {
  // Devuelve "nombre" y "apellido" combinados en una string y separados por un espacio.
  // Ejemplo: "Soy", "Henry" -> "Soy Henry"
  // Tu código:
}

function obtenerSaludo(nombre) {
  // Toma la string "nombre" y concatena otras string en la cadena para que tome la siguiente forma:
  // "Martin" -> "Hola Martin!"
  // Tu código:
}
function esVocal(letra){
  //Escribe una función que reciba una letra y, si es una vocal, muestre el mensaje “Es vocal”. 
  //Verificar si el usuario ingresó un string de más de un carácter, en ese caso, informarle 
  //que no se puede procesar el dato mediante el mensaje "Dato incorrecto".
  // Si no es vocal, tambien debe devolver "Dato incorrecto".
  // Escribe tu código aquí
  }
------------------------------------------------------------------------------------------------------------------------------------
en este codigo no se cual sea el error pero me salta que esta mal resuelto:
function devolverString(str) {
  // "Return" la string provista: str
  // Tu código:
  return str;
}

**Comment by criszam99** [comment]
Lic. Cristina del V. Zamorano
[image: image.png]
